message_ix/model/MESSAGE/model_core.gms

```diff
@@ -921,23 +923,51 @@ TOTAL_CAPACITY_BOUND_LO(node,inv_tec,year)$( is_bound_total_capacity_lo(node,inv
 ***
 * Equation ACTIVITY_BOUND_UP
 * """"""""""""""""""""""""""
-* This constraint provides lower bounds of a technology activity by mode, summed over all vintages.
-*
+* This constraint provides upper bounds of a technology activity, summed over
+* all vintages. If the mode is 'all', then this constraint is applied across all
```

Comment:
This docstring is still pointing to the `'all'` option which was now moved to an own constraint.